assignments/objects.js

```diff
@@ -18,27 +18,70 @@ const example = {
 }
 
 // Write your intern objects here:
+const firstIntern = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F",
+}
+
+const secondIntern = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": "Hello, my name is Kennan!",
+}
+
+const thirdIntern = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M",
+}
 
+const fourthIntern = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M",
+}
 
+const fifthIntern = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  multiplyNums: function(num1, num2) {
+    return num1 * num2
+  }
+}
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(firstIntern.name)
 
 // Kennan's ID
+console.log(secondIntern.id)
 
 // Keven's email
+console.log(secondIntern.email)
 
 // Gannie's name
+console.log(fourthIntern.name)
 
 // Antonietta's Gender
+console.log(fifthIntern.gender)
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
```

Comment:
you could have done a console.log(kennan.speak()); since we know that the second intern has a name key of kennan. You could have also made a function rather than putting the function directly in the object, like this
kennan.speak = function() {
  return "Hello, my name is Kennan!";
}
and then console.log after that